api-controllers/ServiceFabrikAdminController.js

```diff
@@ -735,6 +736,33 @@ class ServiceFabrikAdminController extends FabrikBaseController {
         .send(body));
   }
 
+  createUpdateConfig(req, res) {
```

Comment:
why is create and update clubbed into 1 ? For all other entities we have separate functions right ?

PUT - representing update.
POST - representing create.

Reply:
It is the REST standard we have followed in all API's in the broker. Hence even in this case the same should hold good.

@jagadish-kb : please share your thoughts here.

Reply:
@kamath-prasad especially for this config use case i would recommend merging create/update into one because generally if someone calls a PUT and resource does not exist, one must throw error saying Resource Not Found. Similarly for create if it already exists, we must throw error. Since the configs are something we might toggle, if we go with strict Create & Update, then invokers must first fetch and then do POST or PUT. Hence to keep things simple, it will always be PUT from consumer perspective and merging both Create/Update into one.

Reply:
Ok .. Fine ..